WordPress/gutenberg
# [RNMobile] Remove end-to-end test of Gallery block

## Description


Currently, we only have a single end-to-end test for the Gallery block: `should be able to add a gallery block`. With the introduction of integration tests for this block ([reference](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/gallery/test/index.native.js)), we're already covering the block addition test case, so we no longer require this end-to-end test.

## Testing Instructions




- Observe that all PR check pass in this PR and Gutenberg Mobile PR (https://github.com/wordpress-mobile/gutenberg-mobile/pull/4531).

## Screenshots 
N/A

## Types of changes




Remove unneeded end-to-end test.

## Checklist:
